src/main/kotlin/ru/hse/dynamomock/service/AWSDynamoDBMockService.kt

```diff
@@ -251,27 +258,99 @@ class AWSDynamoDBMockService(private val storage: DataStorageLayer) {
                 storage.updateItem(it)
             }
         }
-        deleteItemRequests.forEach{ storage.deleteItem(it) }
+        deleteItemRequests.forEach { storage.deleteItem(it) }
 
         return BatchWriteItemResponse.builder()
             .build()
     }
 
+    fun loadCSV(fileName: String, tableName: String) {
+        val allowedTypes = ImportableType.values().map { e -> e.name }
+        val header = mutableListOf<Pair<String, String>>()
+        val rows = mutableListOf<List<String>>()
+        csvReader {
+            delimiter = ';'
+            quoteChar = '"'
+        }.open(fileName) {
+            readAllAsSequence().forEach { row ->
+                rows.add(row)
+            }
+        }
+        val firstRow = rows.removeFirstOrNull() ?: throw AWSMockCSVException("The file is empty")
+        firstRow.forEach {
+            val value = it.split("|").map{ v -> v.trim() }
+            if (value.size != 2) {
+                throw AWSMockCSVException("Wrong value format. Use <column_name>|<type>")
+            }
+
+            val (columnName, type) = value.zipWithNext().firstOrNull() ?: throw AWSMockCSVException("Wrong value format. Use <column_name>|")
+            if (type !in allowedTypes) {
+                throw AWSMockCSVException("Function scanItems supports only S, N, NS, SS, NULL, BOOL types right now")
+            }
+            header.add(columnName to type)
+        }
+
+        rows.forEach {
+            val item = it.mapIndexed { index, element ->
+                val (columnName, type) = header[index]
+                columnName to when (ImportableType.valueOf(type)) {
```

Comment:
Looks like a util function for `ImportableType`. Should we move it to the definition of `ImportableType`? IMHO.